radix-engine/tests/metering.rs

```diff
@@ -146,5 +146,23 @@ fn test_grow_memory_out_of_cost_unit() {
     let receipt = test_runner.execute_manifest(manifest, vec![]);
 
     // Assert
-    assert_invoke_error!(receipt.result, InvokeError::MeteringError { .. })
+    assert_invoke_error!(receipt.result, InvokeError::CostingError { .. })
+}
+
+#[test]
+fn test_total_cost_units_consumed() {
+    // Arrange
+    let mut test_runner = TestRunner::new(true);
+
+    // Act
+    let (pk1, _, account1) = test_runner.new_account();
+    let (_, _, account2) = test_runner.new_account();
+    let manifest = ManifestBuilder::new()
+        .withdraw_from_account(RADIX_TOKEN, account1)
+        .call_method_with_all_resources(account2, "deposit_batch")
+        .build();
+    let receipt = test_runner.execute_manifest(manifest, vec![pk1]);
+
+    // Assert
+    assert_eq!(237268, receipt.cost_units_consumed);
```

Comment:
This seems to be a fragile test given any change in the fee table?

Reply:
That's true. The idea was to capture the whole output to avoid any regression issue.  A better solution is to use an expression instead of fixed number to make it less fragile, bug given the size of the wasm, it's hard to do so. 

Thinking I will use the a manually crafted wasm for this test.